src/features/cameras/DeleteCameraAlert.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { red } from '@radix-ui/colors';
+import { styled } from '../../theme/stitches.config.js';
+import Button from '../../components/Button.jsx';
+import { ButtonRow } from '../../components/Form.jsx';
+import {
+  clearDeleteCameraTask,
+  deleteCamera,
+  fetchTask,
+  selectDeleteCameraLoading,
+} from '../tasks/tasksSlice.js';
+import { SimpleSpinner, SpinnerOverlay } from '../../components/Spinner.jsx';
+import { selectSelectedCamera, setSelectedCamera } from '../projects/projectsSlice.js';
+import {
+  fetchCameraImageCount,
+  selectCameraImageCount,
+  selectCameraImageCountLoading,
+  clearCameraImageCount,
+  selectDeleteCameraAlertStatus,
+  setDeleteCameraAlertStatus,
+} from './wirelessCamerasSlice.js';
+import { ASYNC_IMAGE_DELETE_BY_FILTER_LIMIT } from '../../config.js';
+import {
+  Alert,
+  AlertPortal,
+  AlertOverlay,
+  AlertContent,
+  AlertTitle,
+} from '../../components/AlertDialog.jsx';
+import { DeleteImagesProgressBar } from '../images/DeleteImagesProgressBar.jsx';
+
+const BoldText = styled('span', {
+  fontWeight: '$5',
+});
+
+const DeleteCameraAlert = () => {
+  const deleteCameraLoading = useSelector(selectDeleteCameraLoading);
+  const selectedCamera = useSelector(selectSelectedCamera);
+  const imageCount = useSelector(selectCameraImageCount);
+  const imageCountLoading = useSelector(selectCameraImageCountLoading);
+  const alertState = useSelector(selectDeleteCameraAlertStatus);
```

Comment:
I touch on this in the comment below but I would use some more descriptive & consistent names here. "State" and "status" are vague IMO.